Classes/EventStore/Storage/Doctrine/DoctrineEventStorage.php

```diff
@@ -298,7 +304,10 @@ public function setup(): Result
      */
     private function createEventStoreSchema(): Schema
     {
-        $schema = new Schema();
+        $defaultConfig = new SchemaConfig();
+        $defaultConfig->setDefaultTableOptions($this->defaultFlowDatabaseConfiguration);
```

Comment:
I fail to find some documentation about this. Is `setDefaultTableOptions` really expected to get an array with driver database etc options?
Apart from that we should probably not apply the Neos.Flow default backend options but the one configured for the corresponding store (merged with the defaults, like we do in the ConnectionFactory)

Reply:
Oh you are right. Good point!

Reply:
The `defaultTableOptions` are directly given to the Table, so they actually reflect what you can set via `@ORM\Table(options={"collate"="utf8_swedish_ci"})` - The only thing I could find to suggest that this can be provided with the backendOptions is https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Platforms/MySqlPlatform.php#L436
- basically you either provide a property `table_options`, then that will be used or else the optios will be built from the other options like `charset`, `collate`, etc.